non-wheel/httpsrv/httpsrv.py

```diff
@@ -409,12 +409,24 @@ def delete_with_qzui_api(name):
 
 
 def rename(name, newname):
-    print ('rename')
+    print('rename')
     # Could not find a command for this one. Only idea is to use CHGPF or some command to change the member name
     # rerun command to update update apache configuration path
     # Then mv directory to rename directory
 
 
+def start(name):
+    os.system("system 'STRTCPSVR SERVER(*HTTP) HTTPSVR(" + name.upper() + ")'")

```

Comment:
I thought I could just write:

```python
os.system('STRTCPSVR SERVER(*HTTP) HTTPSVR(' + name.upper() + ')')
```

The above wouldn't work, though. It seems to be running system commands under bash instead of IBM i level.